src/content/docs/logs/logs-context/configure-logs-context-apm-agents.mdx

```diff
@@ -45,30 +45,24 @@ Don't spend extra time trying to narrow down all your logs from different parts
-
-For detailed information, see the logs-in-context procedures for:
-
-* [C SDK](/docs/logs/enable-log-management-new-relic/configure-logs-context/c-sdk-configure-logs-context)
-* [Go](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-go/)
-* [Java](/docs/logs/enable-log-management-new-relic/configure-logs-context/java-configure-logs-context-all/)
-* [.NET](/docs/logs/enable-log-management-new-relic/configure-logs-context/net-configure-logs-context-all/)
-* [Node.js](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-nodejs/)
-* [PHP](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-php/)
-* [Python](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-python/)
-* [Ruby](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-ruby/)
-* [Infrastructure monitoring agent](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent/)
+The process to enable logs in context is basically the same, regardless of whether you're setting up an APM agent or the infrastructure agent:
+
+1. Make sure you have already [set up logging in New Relic](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/enable-log-management-new-relic). This includes configuring a supported log forwarder that collects your application logs and extends the metadata that is forwarded to New Relic.
+2. For enabling this for APM: make sure you have a supported APM agent version for your app, and enable [distributed tracing](/docs/distributed-tracing/concepts/introduction-distributed-tracing/). For specific instructions, select your agent: 
+  * [C SDK](/docs/logs/enable-log-management-new-relic/configure-logs-context/c-sdk-configure-logs-context)
+  * [Go](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-go/)
+  * [Java](/docs/logs/enable-log-management-new-relic/configure-logs-context/java-configure-logs-context-all/)
+  * [.NET](/docs/logs/enable-log-management-new-relic/configure-logs-context/net-configure-logs-context-all/)
+  * [Node.js](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-nodejs/)
+  * [PHP](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-php/)
+  * [Python](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-python/)
+  * [Ruby](/docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-ruby/)
+3. For infrastructure monitoring: [enable logs in context for the infrastructure agent](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent).
+4. Look for your logs in context in New Relic One. See [What's next?](#what-next)
```

Comment:
The reason why I separated out the list of agent LiC xrefs as bullets is because, my intention was that people skim the steps and get a sense of what to do, then they feel confident to go to their specific agent. But I understand why you changed this, so [shrug] :-)

Also I suggest changing your step 4 to "4. See [log data in the context of your apps and hosts](#what-next) in New Relic One." But if you prefer to keep as is, be sure to add a period to the end of line 65.

Reply:
i don't have any strong opinions on this and don't fully understand the procedure so feel free to edit/change after I put this out.